doc/data/messages/m/misplaced-bare-raise/details.rst

```diff
@@ -1 +0,0 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
A bare raise statement will re-raise the last active exception in the current scope. If the "raise" statement is not in an ``except`` or ``finally`` block, a RuntimeError will be raised instead.